docs/azdata/install/deploy-install-azdata-linux-package.md

```diff
@@ -25,7 +25,7 @@ For Linux distributions with `apt` there is a package for the `azdata-cli`. The
 ## Install with apt
 
 >[!IMPORTANT]
-> The RPM package of the `azdata-cli` depends on the python3 package. On your system, this may be a Python version which predates the requirement of *Python 3.6.x*. If this poses an issue for you, find a replacement python3 package or follow the manual install instructions that use [`pip`](../install/deploy-install-azdata-pip.md).
+> The DEB package of the `azdata-cli` depends on the python3 package. On your system, this may be a Python version which predates the requirement of *Python 3.6.x*. If this poses an issue for you, find a replacement python3 package or follow the manual install instructions that use [`pip`](../install/deploy-install-azdata-pip.md).
```

Comment:
```suggestion
> The Debian package of `azdata-cli` depends on the `python3` package. On your system, this may be a Python version which predates the requirement of *Python 3.6.x*. If this poses an issue for you, find a replacement `python3` package or follow the manual installation instructions that use [`pip`](../install/deploy-install-azdata-pip.md).
```

Reply:
@derdanu If you don't mind merging the suggested changes, we can merge this pull request, thanks!